addons/event/static/src/client_action/event_registration_summary_dialog.xml

```diff
@@ -3,60 +3,77 @@
-                        The registration must be paid
-                    </div>
                 </div>
             </div>
-            <div id="registration_information" class="row">
+            <div class="row fs-1">
+                <div class="col-lg-12 d-flex align-items-baseline gap-3">
+                    <t t-if="registration.name" t-out="registration.name"/>
+                    <t t-else="" t-out="'Guest #' + registration.id"/>
+                    <t t-if="registration.sale_status_value">
+                        <span class="badge rounded-pill" t-att-class="registration.has_to_pay ? 'text-bg-danger' : 'text-bg-success'">
+                            <t t-out="registration.sale_status_value"/>
+                        </span>
+                    </t>
+                </div>
+            </div>
+            <div id="registration_information" class="row mt-4">
                 <div class="col-lg-12">
-                    <table class="table table-striped">
+                    <table class="table table-striped fs-4">
+                        <tr t-if="registration.company_name"><td>Company</td><td><t t-out="registration.company_name"/></td></tr>
                         <tr><td>Event</td><td><t t-out="registration.event_display_name"/></td></tr>
-                        <tr t-if="!registration.event_id and registration.company_name"><td>Company</td><td><t t-out="registration.company_name"/></td></tr>
-                        <tr><td>Name</td><td><t t-out="registration.name"/></td></tr>
-                        <tr t-if="registration.ticket_name"><td>Ticket</td><td><t t-out="registration.ticket_name"/></td></tr>
-                        <tr t-if="registration.sale_status_value"><td>Payment</td><td><t t-out="registration.sale_status_value"/></td></tr>
-                        <tr t-if="registration.registration_answers">
-                            <td>Answers</td>
-                            <td>
-                                <span t-foreach="registration.registration_answers" t-as="registration_answer" t-key="registration_answer_index"
-                                    t-attf-class="o_tag o_tag_badge_text o_tag_color_#{registration_answer_index % 10} badge rounded-pill p-1 me-1 mb-1"
-                                    t-out="registration_answer"/>
-                            </td>
-                        </tr>
+                        <tr t-if="registration.ticket_name"><td>Ticket Type</td><td><t t-out="registration.ticket_name"/></td></tr>
+                        <t t-if="registration.registration_answers">
+                            <tr t-if="registration.registration_answers.length > 0">
+                                <td class="d-flex">Answers</td>
+                                <td>
+                                    <div class="d-flex flex-wrap p-0">
+                                        <span t-foreach="registration.registration_answers" t-as="registration_answers" t-key="registration_answers_index"
```

Comment:
`t-as="registration_answer"` ? since it is one of the answers I guess? same for index